yorkie/metrics/metrics_server.go

```diff
@@ -0,0 +1,58 @@
+package metrics
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+
+	"github.com/yorkie-team/yorkie/pkg/log"
+	"github.com/yorkie-team/yorkie/pkg/version"
+)
+
+type Config struct {
+	Port int
+}
+
+type Server struct {
+	conf *Config
+}
+
+func NewServer(conf *Config) (*Server, error) {
+	return &Server{
+		conf: conf,
+	}, nil
+}
+
+func (s *Server) listenAndServe() error {
+	go func() {
+		http.Handle("/metrics", promhttp.Handler())
+		if err := http.ListenAndServe(fmt.Sprintf(":%d", s.conf.Port), nil); err != nil {
+			log.Logger.Error(err)
+		}
```

Comment:
When starting the agent, it would be good to print the metrics port as well.

```
❯ ./bin/yorkie agent
2020-08-17T11:29:44.420+0900	INFO	connected, URI: mongodb://localhost:27017, DB: yorkie-meta
2020-08-17T11:29:44.420+0900	INFO	serving API on 9090
```